atlp-rwanda/atlp-pulse-fn
# Add Processing State on Login Button as soon as User Clicks on it

# PR Description
Thi **PR** solves processing state on **login user button**

# How has this been tested?

- Visit **deployment preview** of this **PR**
- Go to `/users/login`
- Enter **credentials**
- As soon as you click on login button, the **processing state** should be triggered imidiately
